### ISSUE TITLE
Engelsk språk

### ISSUE THREAD

**Issue by gselvag** [Puzzlepart/prosjektportalen365#315]
Does the latest version support English language tenant, with English content?

**Comment by tarjeieo** [comment]
No, english version isn't ready yet I'm afraid. I don't think this should require too much effort tbh, but currently not prioritized on the roadmap.

**Comment by tarjeieo** [comment]
We're considering english version as an effort within 2021 H1 - and are looking for financial contributors. Something you're willing to participate in @gselvag ?

**Comment by Cheesydave** [comment]
How much would it cost (ish) to get it for english version? Could put in some legwork for translating if that is needed.

**Comment by David567899** [comment]
We are interested in funding this for the english version, depending on cost. How much do you think it would cost?
